sql/updates/latest-dev.sql

```diff
@@ -22,3 +22,59 @@ BEGIN
     END IF;
 END
 $BODY$;
+
+
+ALTER TABLE  _timescaledb_catalog.continuous_agg
+    ADD COLUMN  ignore_invalidation_older_than BIGINT NOT NULL DEFAULT BIGINT '9223372036854775807';
```

Comment:
is this value valid for all systems i.e.32 and 64 bit?

Reply:
yes. The max values for BIGINT doesn't change